### DISCUSSION TITLE
GPS data automatically removed

### DISCUSSION THREAD

**Discussion by Zyztem** [diced/zipline#225] [Ideas]
Automatically remove GPS data from images

**Comment by diced** [comment]
Possible library to use: https://github.com/xoeye/gps-metadata-remover

**Reply by axisiscool** [reply]
> Possible library to use: https://github.com/xoeye/gps-metadata-remover

Stop using libraries for something that is so simple to re-create. You can easily make the functions that it has faster.

**Reply by superintendent2521** [reply]
then make a pr with the updates

**Reply by rpxs** [reply]
is that a ong fr or a fr ong?

**Reply by superintendent2521** [reply]
thats a "you sassy fuck if u want to do it do it you self" mood

**Reply by rpxs** [reply]
@diced I would like to request you to ban @RhysyPiece360 from any further discussion in this repository, I do not feel welcomed at all by his comments in this repository and feel that his contributions do not follow the Contributor Code of Conduct. Please conduct a thorough review and get back to me with your results.

**Reply by superintendent2521** [reply]
do you not detect the ounce of sarcasm i added to that said sentence,

**Reply by superintendent2521** [reply]
also, L bozo

**Comment by diced** [comment]
Would it be a good idea to also have a way to show exif metadata with each supported file type?

**Comment by diced** [comment]
added in https://github.com/diced/zipline/commit/6457680065af974232b10dee8e6135a32ef9ff97

https://zipline-trunk.diced.tech/docs/config/exif

**Comment by superintendent2521** [comment]
will there be a way to remove whatever exif metadata with the file types. by just setting it to redacted or somthing

**Reply by diced** [reply]
I can make a config variable like `EXIF_REMOVE_TAGS` and you can provide what to remove like `Model,etc,etc,idk` and it will remove, if that's what you mean?

**Reply by superintendent2521** [reply]
the buttons next to copy value and copy name, having one that remove certain exif metadata would be nice, if thats not possible, then EXIF_REMOVE_TAGS would make sense
